packages/commentor/PKGBUILD

```diff
@@ -0,0 +1,29 @@
+pkgname=commentor
```

Comment:
you are missing the disclaimer comments https://github.com/BlackArch/blackarch-pkgbuilds/blob/66259a614509ab1f78075bc897d3cb10d832ed13/PKGBUILD-generic#L1-L2